core/foundation/inc/ROOT/RSpan.hxx

```diff
@@ -14,21 +14,8 @@
 #ifndef ROOT_RSpan
 #define ROOT_RSpan
 
-#include "RConfigure.h"
-
-#ifdef R__HAS_STD_SPAN
-
-#include <span>
-
-#elif defined(R__HAS_STD_EXPERIMENTAL_SPAN)
-
-#include <experimental/span>
-namespace std {
-  using template<class T> span = experimental::span<T>;
-
-  // TODO: using make_view() overloads
-}
-
+#ifdef __cpp_lib_span
```

Comment:
You may need to include some header to make sure this definition is visible. Can we use `__has_include` instead?

Reply:
Ah, I see you do it in a later commit. May be better to squash the two.